modules/microshift-firewall-add-services.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-firewall.adoc
+
+:_mod-docs-content-type: Procedure
+[id="microshift-firewall-add-services_{context}"]
+= Adding services to open ports
+
+On an {microshift-short} instance, you can open services on ports using the `firewall-cmd` command. 
+
+.Procedure 
+
+. Optional: You can view all predefined services in firewalld by running the following command
++
+[source,terminal]
+----
+$ sudo firewall-cmd --get-services
+----
+
+. To open a service you want on a port, run the following example command: 
```

Comment:
I'm probably overthinking this. We're saying "To open a service you want on a port. . .", but then in the command we're not specifying a port. I would think based on the command description that we'd be specifying a port. Is it going to be enabled on the default port? 

So maybe we could consider revising this command description (or maybe not. Maybe it's fine. I dunno!). 

```suggestion
"To enable a service, for example `mdns`, on the default port, run the following command:
```
Or maybe consider some variation of that? 

Or it could be as simple as the following, and in your example we're using the default port. But then how might we specify a specific port? hmm. 

```suggestion
. To open a service that you want on your default port, run the following example command: 
```

Reply:
Yeah that part was kinda confusing to understand when I was first researching this 🙃. I like these suggestions, thank you!